### DISCUSSION TITLE
earapp device limit

### DISCUSSION THREAD

**Discussion by mrjkjxyz** [engageub/InternetIncome#185] [Q&A]
Is it true that the earapp device limit is 10? But I was allowed to add 15 devices.

**Comment by hieunguyen8794** [comment]
nolimit, but suspended when use docker, i add 20 device, first widthdraw is susess, but second time is banned :D

**Reply by mrjkjxyz** [reply]
So it's best not to exceed 10 devices when using Docker? 10 devices should be fine, right?

**Reply by hieunguyen8794** [reply]
nothing is certain, everything is flexible, until they block :v
